### PR TITLE
Add black in RO mode to lint GH action

### PR DESCRIPTION
**Pull request by Bouni**
As discussed in #77 I've added `black` to the lint CI job.

Info: https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#writeback-and-reporting

### CODE REVIEW

**Comment by Bouni** [comment]
@kbabioch Is there a reason for installying pytest within the pylint job?: https://github.com/Bouni/python-luxtronik/blob/00b3ee8dedade655d8b349b371ee4ab164c5ebd3/.github/workflows/lint.yml#L59-L60

**Comment by kbabioch** [comment]
> @kbabioch Is there a reason for installying pytest within the pylint job?:
> 
> https://github.com/Bouni/python-luxtronik/blob/00b3ee8dedade655d8b349b371ee4ab164c5ebd3/.github/workflows/lint.yml#L59-L60

Without it, `pylint` would complain about an import error, see: 8dfeffb90b19a24c5a13a0476de359338e19730c

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -28,6 +28,22 @@ jobs:
         run: pip install --user yamllint
       - name: Run yamllint
         run: yamllint .
+  black:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10", "3.11"]
```
Comment (kbabioch) [inline .github/workflows/lint.yml:35]: Not sure, do we really need to run this with multiple versions of Python?

Comment (Bouni) [reply .github/workflows/lint.yml:35]: I don't think so, in my opinion having only 3.11 is just fine!

Comment (Bouni) [reply .github/workflows/lint.yml:35]: @kbabioch Any thoughts on my last question about pylint?

Comment (kbabioch) [reply .github/workflows/lint.yml:35]: I'm fine with only having 3.11 for linting as well. Sorry for the late reply, somehow I ran into "out of time" errors ;-).

**Comment by Bouni** [comment]
@kbabioch Do you know, is there a way to run actions just once and not for push and pull_request on every commit?

**Comment by kbabioch** [comment]
> @kbabioch Do you know, is there a way to run actions just once and not for push and pull_request on every commit?

No, unfortunately that seems to be not totally trivial. Had looked into this some weeks ago, but couldn't figure it out quickly.

**Pull request merged**